packages/@lwc/synthetic-shadow/src/faux-shadow/element.ts

```diff
@@ -116,6 +344,53 @@ function lastElementChildGetterPatched(this: ParentNode) {
     return children.item(children.length - 1) || null;
 }
 
+defineProperties(HTMLElement.prototype, {
```

Comment:
why is this here and not in html-element.ts file?

Reply:
missed it! good catch!